jackal_base/src/jackal_diagnostic_updater.cpp

```diff
@@ -227,6 +243,62 @@ void JackalDiagnosticUpdater::navsatCallback(const nmea_msgs::Sentence::ConstPtr
   }
 }
 
+#if defined(_WIN32)
+void JackalDiagnosticUpdater::wirelessMonitorCallback(const ros::TimerEvent& te)
+{
+  std_msgs::Bool wifi_connected_msg;
+  wifi_connected_msg.data = false;
+
+  // Get system structure of interface IP addresses.
+  ULONG flags = GAA_FLAG_INCLUDE_PREFIX;
+
+  // check for both IPv4 and IPv6
+  ULONG family = AF_UNSPEC;
+
+  // pre-allocate 15 KB of buffer is sufficient for most computers
+  int length = static_cast<int>(std::ceil(15e3 / sizeof(IP_ADAPTER_ADDRESSES)));
+  auto addresses = std::make_unique<IP_ADAPTER_ADDRESSES[]>(length);
```

Comment:
Does `std::make_unique` need C++14 enabled in the CMakeLists.txt?  Melodic/Bionic targets it but Kinetic/Xenial targets C++11.  However, it looks like ros-win is only for Melodic.

Reply:
@tonybaltovski Would it be possible for you to create a melodic branch? We can then resubmit the pull request.

Reply:
@ooeygui updated to melodic branch.